android/daemon/tunnel/libwg-go/api-android.go

```diff
@@ -224,4 +224,10 @@ func wgVersion() *C.char {
 	return C.CString("unknown")
 }
 
+//export wgGetLatestHandle
+func wgGetLatestHandle() int32 {
+	return int32(len(tunnelHandles) -1)
```

Comment:
What happens if `len(tunnelHandles)` is zero?

Reply:
Any value below 0 is considered "off"" / "wireguard-returned-an-error"